backend/restApi.js

```diff
@@ -0,0 +1,116 @@
+  const gameIdElements = ["", "", ""].map((x) => {
```

Comment:
Is this really the best way of doing this in js...

Reply:
After discussion we'll keep this as it's clear what it's doing at least